include/cpp2util.h

```diff
@@ -741,7 +741,7 @@ auto to_string(std::string const& s) -> std::string const&
 template<typename T>
 auto to_string(std::optional<T> const& o) -> std::string {
     if (o.has_value()) {
-        return std::to_string(o.value());
+        return to_string(o.value());
```

Comment:
This opens up ADL. For now, it should be
```suggestion
        return cpp2::to_string(o.value());
```

Reply:
fixed